types_windows.go

```diff
@@ -0,0 +1,24 @@
+//go:build windows
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package cni
+
+const (
+	DefaultNetDir = "C:\\Program Files\\containerd\\cni\\conf"
+	DefaultCNIDir = "C:\\Program Files\\containerd\\cni"
```

Comment:
[CRI](https://github.com/containerd/containerd/blob/main/pkg/cri/config/config_windows.go#L32) uses `C:\Program Files\containerd\cni\bin` for this path. Should we do the same here to be consistent?

Reply:
IMO let's change to `bin` and then check in :)

Reply:
Done deal!